frontend/src/Activity/History/Details/HistoryDetails.js

```diff
@@ -224,6 +251,113 @@ function HistoryDetails(props) {
       </DescriptionList>
     );
   }
+
+  if (eventType === 'albumImportIncomplete') {
```

Comment:
Should we add any other data to this event detail such as indexer or link as we provide on a successful event.